### PR TITLE
Basic tests

### PR DESCRIPTION
**Pull request by fjsj**
Based on configs from https://github.com/carltongibson/django-filter

### CODE REVIEW

File: admin_totals/admin.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # coding=utf-8
-from __future__ import print_function, unicode_literals, division
+from __future__ import division, print_function, unicode_literals
```
Comment (fjsj) [inline admin_totals/admin.py:2]: isort made this

File: admin_totals/admin.py [hunk 2]
```diff
@@ -13,7 +13,8 @@ def get_results(self, *args, **kwargs):
             list_totals = dict(self.model_admin.list_totals)
             for field in self.list_display:
                 if field in list_totals:
-                    self.aggregations.append(self.result_list.aggregate(agg=list_totals[field](field))['agg'])
+                    self.aggregations.append(
+                        self.result_list.aggregate(agg=list_totals[field](field))['agg'])
```
Comment (fjsj) [inline admin_totals/admin.py:17]: for flake8 to be happy

File: admin_totals/templatetags/admin_totals.py [hunk 3]
```diff
@@ -1,7 +1,9 @@
 # coding=utf-8
-from __future__ import print_function, unicode_literals, division
+from __future__ import division, print_function, unicode_literals
 
-from django.contrib.admin.templatetags.admin_list import result_headers, result_hidden_fields, results
+from django.contrib.admin.templatetags.admin_list import (result_headers,
+                                                          result_hidden_fields,
+                                                          results)
```
Comment (fjsj) [inline admin_totals/templatetags/admin_totals.py:6]: isort made this

**Comment by douwevandermeij** [comment]
Wow, great! Thanks for your contribution!
I'll read through the changed files and merge it later on.

**Comment by douwevandermeij** [comment]
Could you also update the readme a bit to explain how to run tests?

**Comment by fjsj** [comment]
@douwevandermeij done. I've added the Travis and Codecov badges to README too, but you need to sign up there and authorize the repo, since I don't have write permission for this repo. LMK if you need any help. 

Also, added a test for the template.

**Comment by douwevandermeij** [comment]
Thanks, great work!

**Pull request merged**